manageiq-operator/deploy/crds/manageiq.org_manageiqs_crd.yaml

```diff
@@ -0,0 +1,166 @@
+          type: string
+        metadata:
+          type: object
+        spec:
+          description: ManageiqSpec defines the desired state of Manageiq
+          properties:
+            appName:
+              description: 'INSERT ADDITIONAL SPEC FIELDS - desired state of cluster
+                Important: Run "operator-sdk generate k8s" to regenerate code after
+                modifying this file Add custom validation using kubebuilder tags:
+                https://book-v1.book.kubebuilder.io/beyond_basics/generating_crd.html
+                Application name used for deployed objects'
+              type: string
+            applicationAdminPassword:
+              description: admin user initial password
+              type: string
+            applicationDomain:
+              description: Domain name for the external route Used for external authentication
+                configuration
+              type: string
+            databaseRegion:
+              description: Application region number
+              type: string
+            databaseSecret:
+              type: string
+            databaseVolumeCapacity:
+              description: Containerized database volume size
+              type: string
+            encryptionKey:
+              type: string
+            httpdCpuRequest:
+              type: string
+            httpdImageName:
+              type: string
+            httpdImageTag:
+              type: string
+            httpdMemoryLimit:
+              type: string
+            httpdMemoryRequest:
+              type: string
+            kafkaSecret:
+              description: kafka deployment information
+              type: string
+            kafkaVolumeCapacity:
+              type: string
+            memcachedCpuRequest:
+              description: memcachedpod deployment information
+              type: string
+            memcachedImageName:
+              type: string
+            memcachedImageTag:
+              type: string
+            memcachedMaxConnection:
+              type: string
+            memcachedMaxMemory:
+              type: string
+            memcachedMemoryLimit:
+              type: string
+            memcachedMemoryRequest:
+              type: string
+            memcachedSlabPageSize:
+              type: string
+            orchestratorCpuRequest:
+              description: ' orchestrator deployment information'
+              type: string
+            orchestratorImageName:
+              type: string
+            orchestratorImageNamespace:
+              type: string
+            orchestratorImageTag:
+              type: string
+            orchestratorMemoryLimit:
+              type: string
+            orchestratorMemoryRequest:
+              type: string
+            postgresqlCpuRequest:
+              description: postgres database pod deployment information
```

Comment:
These descriptions look either incorrect or just incomplete. I would just remove them for now.

Reply:
@gyliu513 where does this come from? Can we remove these descriptions? They don't really make sense.

Reply:
@gyliu513 still waiting on this one as well.

Reply:
Done. This was autogenerated from the comments in manageiq_types.go, let me remove it for now.